README.md

```diff
@@ -84,28 +89,46 @@ For more information on Jenkins and Docker setup read the corresponding document
 
 # Building & Running the application locally
 
-First, start the zookeeper, and kafka services that is described in the ```doc/runKafka.txt``` file.
+First, start the zookeeper, and kafka services that is described in the ```doc/runKafka.md``` file.
 
 Then, to build the application, issue the following command in the parent project folder
 
 ```bash
 mvn clean install
 ```
 
-After this, to run the project, issue the following command from the folder of starter module:
-### Prerequisite of run:
-    1. Requires to set up an environment variable for encryptor MASTER_PASSWORD, POSTGRES_PWD, POSTGRES_USR (as the VPS docker were set up)
+## Connecting to Postgres
+
+There are two options to connect to Postgres
+
+1. Run the Postgres on the VPS and connect to that - as described in the ```doc/runPostgres.md``` file.
+1. Install Postgres locally and set the ```spring.datasource.url``` system property to point to your local database: ```-Dspring.datasource.url=jdbc:postgresql://localhost:anyport/dbname```
+
+## Encrypting passwords before running the application:
+    1. Requires to set up an environment variable for encryptor BONNIE_MASTER_PASSWORD, POSTGRES_PWD, POSTGRES_USR (The ```POSTGRES_*``` values are the ones specified by your choice of postgres installation)
     2. Generate credential using the following command:
     bash:   - mvn jasypt:encrypt-value -Djasypt.encryptor.password=$BONNEE_MASTER_PWD -Djasypt.encryptor.algorithm=PBEWithMD5AndDES -Djasypt.plugin.value=$POSTGRES_PWD
     3. Grab the output value of each, (it starts with ENC(..., ) then place into the application properties file 
         e.g: spring.datasource.username=ENC(6KpVjqrPwKvLt/5Cjo2ZHg==)
              spring.datasource.password=ENC(6KpVjqrPwKvLt/5Cjo2ZHg==) 
      mvn spring-boot:run -Dspring-boot.run.arguments="--jasypt.encryptor.password=$BONNEE_MASTER_PWD --spring.datasource.password=ENC(6KpVjqrPwKvLt/5Cjo2ZHg==)"
 
+
+## Running the application
+
+You can run the bonnie application with this command from the ./starter directory:
```

Comment:
`bonnie application` means the backend? or the entire application with forntend and backend?

Reply:
Backend

Reply:
could you please specify the backend in this case here?